sdmetrics/reports/utils.py

```diff
@@ -153,8 +154,8 @@ def make_continuous_column_plot(real_column, synthetic_column, sdtype):
         plotly.graph_objects._figure.Figure
     """
     column_name = real_column.name if hasattr(real_column, 'name') else ''
-    missing_data_real = round((real_column.isna().sum() / len(real_column)) * 100, 2)
-    missing_data_synthetic = round((synthetic_column.isna().sum() / len(synthetic_column)), 2)
+    missing_data_real = round(get_missing_percentage(real_column), 2)
```

Comment:
I think even the rounding part could go in the `get_missing_percentage` function

Reply:
Moved the rounding part inside the function.